apps/sms/services/js/activity/client.js

```diff
@@ -0,0 +1,85 @@
+/* global EventDispatcher,
+          bridge
+*/
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for managing activities.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'activity-service';
+
+  /**
+   * Reference to active bridge client instance.
+   * @type {Client}
+   */
+  var client = null;
+
+  /**
+   * Indicates if there is pending activity request.
+   * @type {boolean}
+   */
+  var hasPendingRequest = false;
+
+  var ActivityClient = {
+    /**
+     * Initialized activity service client bridge.
+     */
+    init() {
+      client = bridge.client(SERVICE_NAME);
+
+      client.on('activity-request', (request) => {
+        hasPendingRequest = true;
```

Comment:
Yeah, I can postError if you want :)

The only my concern about the code that is not used now is that it's confusing when you read it later on, but it's still not used or case it covers, still doesn't exist - I usually spend time to find the case it covers (and eager to remove if can't find :D) :) Just say me what way you prefer!